test/acl.test.js

```diff
@@ -90,6 +90,42 @@ describe('security scopes', function() {
 });
 
 describe('security ACLs', function() {
+  it('supports checkPermission() returning a promise', function(done) {
+    ACL.create({
```

Comment:
hi @superkhau : i am aware of this, i thought you'd like to stick to the origin test file style. as per recent @bajtos PR where the callback style is still used (e.g. [here in `role.test.js`](https://github.com/strongloop/loopback/pull/3146/files#diff-b7e36d904fe1d290934c235affb649ecR308))
i'll modify to use mocha with promises